src/Build/Graph/ProjectGraphNode.cs

```diff
@@ -10,6 +11,7 @@ namespace Microsoft.Build.Experimental.Graph
     /// <summary>
     /// Represents the node for a particular project in a project graph.
     /// </summary>
+    [DebuggerDisplay(@"{ToString()}")]
```

Comment:
You could do this without exposing `ToString()` as public API by calling the method something else; might be worth doing.